deployment/k8s/charts/values.yaml

```diff
@@ -26,6 +28,15 @@ ingress:
   #    hosts:
   #      - titiler.local
 
+extraHostPathMounts: []
+  # - name: map-sources
+  #   mountPath: /map-sources/
+  #   hostPath: /home/ubuntu/map-sources
+  #   readOnly: false
+  #   mountPropagation: HostToContainer # OPTIONAL
+
+imagePullSecrets: []
+
```

Comment:
nitpick: but I like what you did above with `extraHostPathMounts` showing an example of what should go there. Can we add `# -name: regcred` here also please as an example?